examples/exec_delta_housekeeping.py

```diff
@@ -0,0 +1,47 @@
+# Databricks notebook source
+# MAGIC %md
+# MAGIC # Run Delta Housekeeping across multiple tables
+# MAGIC Analysis that provides stats on Delta tables / recommendations for improvements, including:
+# MAGIC - stats:size of tables and number of files, timestamps of latest OPTIMIZE & VACUUM operations, stats of OPTIMIZE)
+# MAGIC - recommendations on tables that need to be OPTIMIZED/VACUUM'ed
+# MAGIC - are tables OPTIMIZED/VACUUM'ed often enough
+# MAGIC - tables that have small files / tables for which ZORDER is not being effective
+# MAGIC
+
+# COMMAND ----------
+
+# MAGIC %pip install dbl-discoverx
+
+# COMMAND ----------
+
+from discoverx import DX
+
+dx = DX()
+
+# COMMAND ----------
+
+# DBTITLE 1,Run the discoverx DeltaHousekeeping operation -generates an output object on which you can run operations
+output = (
+  dx.from_tables(f"{dbutils.widgets.get('catalog')}.*.*")
```

Comment:
You need to define the widget before you can use it.
I usually define a `from_tables` widget where you can put the whole pattern, like this

dbutils.widgets.text("from_tables", "*.*.*")
from_tables = dbutils.widgets.get("from_tables")

Reply:
apologies for the oversight, solved